arm9/lib/include/OS_mutex.h

```diff
@@ -0,0 +1,20 @@
+//
+// Created by red031000 on 2020-05-17.
+//
+
+#ifndef POKEDIAMOND_OS_MUTEX_H
+#define POKEDIAMOND_OS_MUTEX_H
+
+#include "types.h"
+#include "OS_thread.h"
+
+#pragma warn_padding off //apparently needed?
```

Comment:
Is this actually needed?   
We don't build with this warning on anyway.

Reply:
should I remove this then? since during testing I build with -W all